### DISCUSSION TITLE
exception/try-catch-xingneng

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#203] [Announcements]
# exception/try-catch-xingneng

本文详细探讨了try-catch捕获异常在Java编程中是否会影响性能。通过对比实验和性能测试，分析了异常处理与性能的关系，解答了关于try-catch对性能影响的常见疑问。阅读本文，将帮助您在编写代码时更加明智地使用异常处理机制，同时确保程序性能的稳定和优越。

https://tobebetterjavaer.com/exception/try-catch-xingneng.html

**Comment by ljljava** [comment]
不错不错，赞一个。
